primitives/src/v1.rs

```diff
@@ -231,6 +231,11 @@ impl<H: Clone> CommittedCandidateReceipt<H> {
 	pub fn hash(&self) -> CandidateHash where H: Encode {
 		self.to_plain().hash()
 	}
+
+	/// Does this committed candidate receipt corrensponds to the given [`CandidateReceipt`]?
+	pub fn corresponds_to(&self, receipt: &CandidateReceipt<H>) -> bool where H: PartialEq {
```

Comment:
this method is used only for testing in one place, is it worth making it a pub method?

Reply:
Maybe it is useful somewhere else. I did not scan all the code if we already need something similar somewhere else. 

TLDR: Yes.